packer_files/vsphere_centos_template/build.pkr.hcl

```diff
@@ -0,0 +1,13 @@
+packer {
+  required_plugins {
+    vsphere = {
+      source  = "github.com/hashicorp/vsphere"
```

Comment:
I couldn't find this repository. Did you meant to use https://github.com/hashicorp/terraform-provider-vsphere ?

Reply:
It's packer not terraform

Reply:
https://www.packer.io/docs/builders/vsphere